CleverTapSDK/CleverTap.h

```diff
@@ -904,6 +912,8 @@ This sets the CleverTapPushNotificationDelegate.
 - (void)setInAppNotificationDelegate:(id <CleverTapInAppNotificationDelegate> _Nullable)delegate;
 #endif
 
+- (void)setURLDelegate:(id <CleverTapURLDelegate> _Nullable)delegate;
```

Comment:
Please add a concise description of its functionality, followed by any caveats or additional details.